STX.REST.RESTFulSense.Clients/Services/Foundations/HttpExchanges/HttpExchangeService.cs

```diff
@@ -38,12 +38,12 @@ await ValidateOnGetAsync(
                 defaultHttpVersion,
                 defaultHttpVersionPolicy);
 
-           HttpRequestMessage httpRequestMessage =
-                MapToHttpRequest(
-                    httpExchangeRequest: httpExchange.Request,
-                    defaultHttpMethod: defaultHttpMethod,
-                    defaultHttpVersion: defaultHttpVersion,
-                    defaultHttpVersionPolicy: defaultHttpVersionPolicy);
+            HttpRequestMessage httpRequestMessage =
```

Comment:
Remove extra spaces at the beginning

Reply:
These extra spaces are not visible on my nachine
![Uploading image.png…]()